### PR TITLE
Use localization middleware

### PR DESCRIPTION
**Pull request by svick**
Fixes https://github.com/aspnet/MusicStore/issues/800 by forcing the culture to be `en-US` using the localization middleware.

For a non-localized application, I believe this is the correct behavior.

Also, are five different versions of `Startup.cs` really necessary? Couldn't the common parts be extracted to a shared file? I personally had issues with this, when the smoke test kept failing even after I fixed the three `Startup`s in the root directory.

### CODE REVIEW

File: samples/MusicStore/ForTesting/Mocks/StartupOpenIdConnectTesting.cs [hunk 1]
```diff
@@ -113,6 +115,15 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            var supportedCultures = new[] { new CultureInfo("en-US") };
+
+            app.UseRequestLocalization(new RequestLocalizationOptions
```
Comment (pranavkm) [inline samples/MusicStore/ForTesting/Mocks/StartupOpenIdConnectTesting.cs]: Could you add a comment stating why you're forcing `en-US`? Would be helpful for posterity.

Comment (svick) [inline samples/MusicStore/ForTesting/Mocks/StartupOpenIdConnectTesting.cs]: Done.

**Comment by natemcmaster** [comment]
@svick to get CI tests to pass on Linux, you'll need to rebase on the latest changes in dev branch so your PR has this fix: https://github.com/aspnet/MusicStore/commit/0c7c05e1ce3f1d16b2caa605cf0c929bd6a24c6f

**Comment by svick** [comment]
@natemcmaster Done.

**Comment by pranavkm** [comment]
Thanks for the PR @svick

**Pull request merged**